File: TP/TP1/Code/5J-1.cpp
/* Afficher un motif donné */
#include <iostream>

int main()
{
  for (int i = 0; i < 9; ++i) {
    if (i < 5)
      for (int j = 0; j < i + 1; ++j)
        std::cout << "*" << std::flush;
    else
      for (int j = 0; j < 9 - i; ++j)
        std::cout << "*" << std::flush;
    std::cout << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/5J-1.py
#!/usr/bin/python
# Afficher un motif donné
for i in range(9):
    if i < 5:
        for j in range(i + 1):
            print('*', end='')
    else:
        for j in range(9 - i):
            print('*', end='')
    print()  # Ici on affiche un saut de ligne
